policy/microos/rancher.te

```diff
@@ -0,0 +1,87 @@
+policy_module(rancher, 1.0.0)
+
+gen_require(`
+    type container_runtime_t, unconfined_service_t;
+')
+
```

Comment:
I'm sorry to ask but could you now add these policies https://github.com/rancher/rancher-selinux/blob/1384bec84b55e912c7ae8db48c87767faca07b89/policy/centos8/rancher.te#L7-L22 as well?